README.md

```diff
@@ -104,34 +116,24 @@ Every week, the latest version of the data is automatically bulk copied ("ingest
 
 You can check the health of a live deployment of the API by running the live integration tests.
 
-1. Change directory to the `openverse-api`
+1. Run the install recipe:
 
 ```
-cd openverse-api
+just install
 ```
 
 #### On the host
 
-1. Install all dependencies for Openverse API.
+1. Run the tests in a Pipenv subshell.
 ```
-pipenv install
-```
-
-2. Run the tests in a Pipenv subshell.
-```
-pipenv run bash ./test/run_test.sh
+just testlocal
```

Comment:
I got this by trying to run this command:

```sh
error: Justfile does not contain recipe `testlocal`
```

Reply:
Oh hmm I removed it for some reason. Let me see.

Do you know why these tests must run in a "pipenv subshell"?

Reply:
Personally, I run the tests inside the container, we can leave only those instructions for tests.

```shell
docker-compose exec web bash ./test/run_test.sh
```

Reply:
I wonder if it's ever necessary to run it outside of the container 🤔

Reply:
Sometimes, I run the code outside the container when debugging it line by line. Running the tests outside the containers helps in that scenario. I'm open to a different process though if I'm the only one using it like that.

Reply:
Yeah that seems like a reasonable use case. Can't `PyCharm` attach to the in-container interpreter though? I never tried to use the GUI debugger for that though, I always relied on `pdb` 😞 the GUI is so much nicer when it's available. I'll think about some alternative approaches to this problem.

Reply:
Yes, Pycharm can take the interpreter of a container, it's quite flexible in this aspect. However, I can see the value in the local approach in not being tied up to this particular IDE, neither to the use of `just` if that is what is blocking here.

Reply:
@krysal from what I had read in the docs, PyCharm can work with a Docker environment only if it created the container. I wasn't able to get it to connect with the existing `web` container orchestrated by Docker Compose 😢.

Reply:
Hmm, I don't think that's true Dhruv, the docs might be incorrect. At my previous job we were able to get it to connect to the docker container just fine and it was running using docker compose... but maybe things have changed since then.

Reply:
In any case, after making the changes in the latest commits we're able to add back the `testlocal` script 🎉

Reply:
The `testlocal` script works perfectly. So does the regular `test` one.